src/components/dashboard/DashboardPage.jsx

```diff
@@ -80,15 +87,14 @@ const DashboardPage = () => {
           <Tab eventKey="courses" title="Courses">
             {CoursesTabComponent}
           </Tab>
-          <Tab eventKey="programs" title="Programs">
+          <Tab eventKey="programs" title="Programs" disabled={!!contentTypeSet.has('programs')}>
             <ProgramListingPage />
           </Tab>
           {features.FEATURE_ENABLE_PATHWAY_PROGRESS && (
-            <Tab eventKey="pathways" title="Pathways">
+            <Tab eventKey="pathways" title="Pathways" disabled={!!contentTypeSet.has('learnerpathway')}>
```

Comment:
[clarification] Can you explain the logic here a bit more? As is, the "Programs" tab appears to get disabled when there _are_ programs in the highlighted content item(s). Is this the intent? How do these changes impact enterprises who don't have any highlights?

Also, running this locally, I have a highlight set containing both a course and a program. In this logic,`contentTypeSet` contains `Set(2) {'course', 'program'}`. However, the conditional in the `disabled` prop for the "Programs" tab refers to a "programs" string, not singular "program".

[clarification] Even if an enterprise's highlighted content only contains courses, a user who enrolls in a highlighted course associated with a program(s), I _believe_ they are deemed to implicitly be "enrolled" as part of the associated program. In this case, the highlighted content type is only "course" but the resulting API call from `ProgramListingPage` to fetch the user's `getLearnerProgramsList` might actually return a program associated with the user's enrollment in the highlighted course. To confirm, the intent is to hide the "Programs" tab since the highlighted content itself doesn't explicitly include a program?

Reply:
Question 1: If there are no content highlight sets. It will still return an empty set. If a set does contain either 'program'(typo in code now fixed), or 'learnerpathway', it will set the disabled flag to true, to prevent interaction with the individual tabs.

Question 2: resolved typo :) thanks for the catch.

Question 3: Correct the intent is to hide it if it exist explicitly within the content highlight set highlighted contents. 
I was not aware of it returning a program. Maybe an additional conditional that the user does have the `canOnlyViewHighlightSets` enabled, then allow the tabs to be disabled.

Reply:
> Correct the intent is to hide it if it exist explicitly within the content highlight set highlighted contents.

@brobro10000 [clarification] So, if my enterprise has a highlight set containing a course and a program, I'll see both on the search page. Then, I might click on the program card and decide to enroll in the first course within the program. With that logic, on my dashboard, I'd only have "Courses" tab available even though I originally enrolled in the course via the program card in the highlight set?

Wouldn't it be reasonable to assume the learner might want to see the program listing from which they enrolled in the course via the "Programs" tab on their dashboard?

Reply:
So I finally see the flaw in my logic. I also added an additional conditional to only render the logic when the user can only see highlighted content. We might need more clarification on the final implementation. But as of now the corrected is as follows
![Screenshot 2023-01-25 at 3 00 25 PM](https://user-images.githubusercontent.com/82611798/214676552-30bd2f37-6335-40b0-95a5-8b003fe54ba3.png)